app/models/data.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Column, Integer, Date, Time, String, Float
+
+from db.base_class import Base
+
+
+class Data(Base):
+    __tablename__ = 'data'
+    id = Column(Integer, primary_key=True, unique=True, nullable=False)
+    date = Column(Date, nullable=True)
```

Comment:
can all fields be `nullable` ?

Reply:
These are all nullable due to two reasons:
1. The ploutos dataset that @nikoskal provided to be used with this service contains rows with 0 columns present, except the date.
2. In practice it is possible to have measurement readings with no information, that timeslot in essence is empty, save for the datetime field.